unity-renderer/Assets/DCLServices/WearablesCatalogService/WebInterfaceWearablesCatalogService.cs

```diff
@@ -123,6 +123,17 @@ public async UniTask<WearableItem> RequestWearableAsync(string wearableId, Cance
             return await taskResult.Task.AttachExternalCancellation(ct);
         }
 
+        public UniTask<WearableItem> RequestWearableFromBuilderAsync(string wearableId, CancellationToken ct) =>
+            throw new NotImplementedException("Supported by LambdasWearablesCatalogService");
+
+        public UniTask<IReadOnlyList<WearableItem>> RequestWearableCollection(IEnumerable<string> collectionIds,
+            CancellationToken cancellationToken, List<WearableItem> wearableBuffer = null) =>
+            throw new NotImplementedException("Supported by LambdasWearablesCatalogService");
+
+        public UniTask<IReadOnlyList<WearableItem>> RequestWearableCollectionInBuilder(IEnumerable<string> collectionIds,
+            CancellationToken cancellationToken, List<WearableItem> wearableBuffer = null) =>
+            throw new NotImplementedException("Supported by LambdasWearablesCatalogService");
```

Comment:
as well as here. I will probably even prefer upcast with null check then having these Exceptions here. But better to split to different interfaces with respect of clients usages.

Reply:
`WebInterfaceEmotesCatalogService` and `WebInterfaceWearablesCatalogService` are marked as obsolete and could be removed after this implementation. They are only being used to preview wearables & emotes in the backpack v1. This implementation is deprecating that flow. So i think its over-engineering at this point to segregate into new interfaces.